labs/src/main/scala/org/scalalabs/basic/lab03/RecursionPatternMatchingExercise.scala

```diff
@@ -28,40 +28,54 @@ object RecursionPatternMatchingExercise {
    * checkValuesIncrease(Seq(1,2,3)) == true
    * checkValuesIncrease(Seq(1,2,2)) == false
    */
-  def checkValuesIncrease(seq: Seq[Int]): Boolean = {
-    error("fix me")
+  def checkValuesIncrease(seq: Seq[Int]): Boolean = seq match{
+    case s:Seq[_] if s.length < 2 => true
+    case first::second::rest if first < second => checkValuesIncrease(second::rest)
+    case _ => false
   }
   
   /**
    * Group Consecutive values
    * List(1,1,2,3,1,1) -> List(1,1), List(2), List(3), List(1,1)
    */
-  def groupConsecutive[T](in: List[T]): List[List[T]] = {
-    error("fix me")
+  def groupConsecutive[T](in: List[T]): List[List[T]] = in match{
+    case Nil => Nil
+    case first::rest => {
+      (first::rest.takeWhile(_==first))::groupConsecutive(rest.dropWhile(_==first))
+    }
```

Comment:
Good use of builtins (takewhile and dropwhile). My solution did it manually, and was much worse.